docs/examples/data/pandapower/example_simple_input.json.license

```diff
@@ -1,3 +1,3 @@
-SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
 
-SPDX-License-Identifier: MPL-2.0
+SPDX-License-Identifier: MPL-2.0
```

Comment:
please out of convention re-add the whiteline for all files (including the ones that didn't have it before)

Reply:
Included in the latest commit.